REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.3
-Nettle 0.2
-JSON 0.5
-ZMQ 0.3
-Compat 0.7.15
-Conda 0.1.5
+Compat 0.8.0
+Nettle 0.2.2
+JSON 0.5.2
+ZMQ 0.3.2
```

Comment:
for Compat 0.8.0
- `bytestring(::Ptr, ...)` has been replaced with `unsafe_string`

for other packages, keeping the version numbers.